consolelabs/mochi-discord
# feat: earn new list

**What does this PR do?**

-   [x] Mockup UI for `/earn new` cmd and CTA buttons
-   [x] Mockup UI for airdrop detail and CTA buttons

**How to test**

-   Type `/earn new`
-   Select an item in the dropdown to see the detail

**Edge cases**

-   Mockup only

**Media (Loom or gif)**

![Screenshot 2023-06-05 at 11 25 11](https://github.com/consolelabs/mochi-discord/assets/11255278/967955c4-4451-49d1-a41f-6b069d562907)
![Screenshot 2023-06-05 at 11 25 04](https://github.com/consolelabs/mochi-discord/assets/11255278/b27cf125-503b-48d7-bae5-ed458fc76b87)
